src/api/index.js

```diff
@@ -108,6 +108,21 @@ export async function getMentors() {
   return [];
 }
 
+export async function getFavorites() {
+  const {_id: userId} = await getCurrentUser();
+  const res = await makeApiCall(`${paths.USERS}/${userId}/favorites/`);
+  if (res.success) {
+    return res.data.mentors.map(mentor => mentor._id);
+  }
+  return [];
```

Comment:
What about doing:

```
const res = await makeApiCall(`${paths.USERS}/${userId}/favorites/`);
return res.success
    ? res.data.mentors.map(mentor => mentor._id)
    : []
```

And even cleaner:
```
const {success, data} = await makeApiCall(`${paths.USERS}/${userId}/favorites/`);
return success
    ? data.mentors.map(mentor => mentor._id)
    : []
```

And even with better performance: (because we don't really need to `await`.
```
return makeApiCall(`${paths.USERS}/${userId}/favorites/`)
    .then(({success, data}) => success
         ? data.mentors.map(mentor => mentor._id)
        : []
)
```

Reply:
Personally it looks clearer to me..
return after `await` will basically return a Promise which will be resolved by `makeApiCall` result, the same as regular Promise. Isn't?

Reply:
I'm with @moshfeu in this one, I try to avoid ternaries whenever I can, 
a simple `if` is more natural and easier to understand, dumb code is usually better than clever code, it is easier for beginners